src/actions/LogActions.tsx

```diff
@@ -61,23 +65,37 @@ export function showSendLogsModal(): ThunkAction<Promise<void>> {
     const { isConnected } = state.network
     if (!isConnected) return showError(s.strings.network_alert_title)
 
-    Airship.show<string | undefined>(bridge => (
-      <TextInputModal
+    await Airship.show(bridge => <LogsModal bridge={bridge} />)
+  }
+}
+export function showClearLogsModal(): ThunkAction<Promise<void>> {
+  return async (dispatch, _getState) => {
+    Airship.show<string | number | undefined>(bridge => (
+      <ButtonsModal
         bridge={bridge}
-        inputLabel={s.strings.settings_modal_send_logs_label}
-        returnKeyType="send"
-        title={s.strings.settings_button_send_logs}
-        onSubmit={async notes => {
-          await dispatch(prepareLogs(notes))
-          showToast(s.strings.settings_modal_send_logs_success)
-          return true
+        title={s.strings.settings_button_clear_logs}
+        message={s.strings.settings_modal_clear_logs_message}
+        buttons={{
+          yes: {
+            label: 'Clear',
+            type: 'secondary',
+            onPress: async () => {
+              await clearAllLogs()
+              showToast(s.strings.settings_modal_clear_logs_success)
+              return true
+            }
+          },
+          no: {
+            label: 'Cancel',
+            type: 'escape'
+          }
         }}
       />
-    ))
+    )).catch(showError)
```

Comment:
This new function seems to be a copy of the function below. We don't need two copies.

Reply:
Removed the duplicate below since it was the one not being used.